oracle/oracle.go

```diff
@@ -280,6 +280,22 @@ func (o *oracleOps) InspectInstance(instanceID string) (*cloudops.InstanceInfo,
 	}, nil
 }
 
+func (o *oracleOps) GetInstance(displayName string) (interface{}, error) {
+	listInstanceReq := core.ListInstancesRequest{
+		DisplayName:   common.String(displayName),
+		CompartmentId: common.String(o.compartmentID),
+	}
+
+	listInstanceResp, err := o.compute.ListInstances(context.Background(), listInstanceReq)
+	if err != nil {
+		return nil, err
+	}
+	if len(listInstanceResp.Items) == 0 {
+		return nil, fmt.Errorf("no oracle instance found with display name: %s", displayName)
+	}
+	return listInstanceResp.Items[0], nil
```

Comment:
Can we have 2 instances with same displayname?

Reply:
Currently, torpedo uses this API to fetch details of the instance created  by OKE. OKE ensures that all the worker nodes created, have unique display names. In future, if we require to use this API to get details of vanilla compute instances, then we can modify this array indexing.